Dockerfile

```diff
@@ -1,21 +1,24 @@
-#https://github.com/anapsix/docker-alpine-java
-FROM nexus.303net.net:8443/anapsix/alpine-java:8_jdk
+#https://hub.docker.com/layers/library/alpine/3.17.3/images/sha256-b6ca290b6b4cdcca5b3db3ffa338ee0285c11744b4a6abaa9627746ee3291d8d?context=explore
+ARG base=nexus.303net.net:8443/alpine:3.17.3
```

Comment:
This has been tested and works as expected. Results can be found [here](https://jira.integralads.com/browse/CICD-8118?focusedCommentId=1099634&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1099634)